chat-plugins/info.js

```diff
@@ -202,7 +203,10 @@ exports.commands = {
 				}
 			});
 		}
-		if (!results.length) return this.errorReply("No results found.");
+		if (!results.length) {
+			let invalidTarget = net.isIP(target.trim()) === 0 && !target.includes('.');
+			return this.errorReply("No results found" + (invalidTarget ? " because this is not a valid IP or host search quarry." : "."));
```

Comment:
>Are you sure it's quarry and not query?

Yes, it appears so:  http://www.english-for-students.com/What-is-the-difference-between-QUARRY-and-QUERY.html